contracts/LingoNFT.sol

```diff
@@ -37,37 +37,31 @@ contract LingoNFT is ERC721, EIP712, Ownable, ReentrancyGuard {
     }
 
     /// @dev Mapping to store Tier for each token ID
-    mapping(uint256 => Tier) private _tokenTier;
-
-    /// @dev Mapping to store URI for each NFT tier
-    mapping(Tier => string) private _tierURIs;
+    mapping(uint256 => bool) private privateJet;
 
     /// @notice Struct for Mint data
     struct MintData {
         address sender;
         Tier tier;
     }
 
+    string public firstClassURI = "ipfs://QmWsdztWmpXf8hiuGX8p8sXdn6K6J6zWuWWMcgaa6TaP2H";
+    string public privateJetURI = "ipfs://QmPNSZ2jgQtLnk46EaCvcm5WQ31PZDMeCtgtPfbBbtBMsx";
+
     /// @dev Tracks whether an address has minted for a specific tier
     mapping(address => mapping(Tier => bool)) private _hasMinted;
```

Comment:
A mapping is wasteful, use a struct
(If there were more than 32 tiers, you would have to use OpenZeppelin's lib)